src/components/ReactQueryWrapper.ts

```diff
@@ -0,0 +1,31 @@
+import * as APIFunctions from "./APIFunction"
+import { useQuery, useMutation } from "react-query"
+
+interface Comment {
+    userName: string, 
+    repoName: string, 
+    issueNumber: string, 
+    comment: string
+}
+
+export const GetUser = (userName : string) => {
+    return (useQuery(['user'], () => APIFunctions.getUser(userName)))
+}
+
+export const GetRepos = (userName: string) => {
+    return (useQuery(['repos'], () => APIFunctions.getRepos(userName)))
+}
+
+export const GetBranches = (userName: string, repoName: string) => {
+    return (useQuery(['branches'], () => APIFunctions.getBranches(userName, repoName)))
+}
+
+export const GetPullRequests = (userName: string, repoName: string) => {
+    return (useQuery(['pullRequests'], () => APIFunctions.getPullRequests(userName, repoName)))
+}
+
+export const AddComment = () => {
+    return useMutation((newComment: Comment) => {
+    return APIFunctions.addComment(newComment.userName, newComment.repoName, newComment.issueNumber, newComment.comment)
+})}
```

Comment:
```suggestion
    return useMutation((newComment: Comment) => 
    APIFunctions.addComment(newComment.userName, newComment.repoName, newComment.issueNumber, newComment.comment)
)}
```

Also, when your function receives more than 3 arguments a good practice is to change it to an object instead. But in this case maybe it's possible to just send `newComment` and destruct it's properties inside `addComment` function